notebook/notebookapp.py

```diff
@@ -1379,6 +1393,22 @@ def init_logging(self):
         logger.parent = self.log
         logger.setLevel(self.log.level)
     
+    def init_resources(self):
+        if resource is None:
+            self.log.debug('Ignoring min_open_files_limit because the limit cannot be adjusted (for example, on Windows)')
+            return
+
+        old_soft, old_hard = resource.getrlimit(resource.RLIMIT_NOFILE)
+        soft = self.min_open_files_limit
+        hard = old_hard
+        if old_soft < soft:
+            if hard < soft:
+                hard = soft
+            resource.setrlimit(resource.RLIMIT_NOFILE, (soft, hard))
+            self.log.debug(
+                'Raised open file limit: soft {}->{}; hard {}->{}'.format(old_soft, soft, old_hard, hard)
+            )
+
```

Comment:
This strikes me as more of an informational message than debug.  I would also suggest taking an optimistic approach and logging 'Raising open file...' prior to the actual call.  This way, the log will contain the values that were presumably bad and could prove helpful in troubleshooting.

Reply:
Very good point!

Reply:
I went ahead and left it a debug message, but moved it before the action. My thinking is that for most people, this should never be a concern, so don't clutter up their terminal. However, if you are trying to track down an issue, likely you are running with --debug.